example/FamilyHubs.Example/Pages/Examples/FullPages/SingleAutocomplete/SingleAutocomplete.cshtml.cs

```diff
@@ -0,0 +1,55 @@
+using FamilyHubs.SharedKernel.Razor.ErrorNext;
+using FamilyHubs.SharedKernel.Razor.FullPages.SingleAutocomplete;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.Collections.Immutable;
+
+namespace FamilyHubs.Example.Pages.Examples.FullPages.SingleAutocomplete;
+
+public record Dto(long Id, string Name);
+
+public class SingleAutocompleteModel : PageModel, ISingleAutocompletePageModel
+{
+    [BindProperty]
+    public string? SelectedValue { get; set; }
+    public string Label => "Search and select the local authority area this service is in";
+    public string? DisabledOptionValue => (-1).ToString();
```

Comment:
nitpick: Little comment here too as to why `-1`

Reply:
There's XML documentation on the DisabledOptionValue property in the interface to guide consumers of the interface (which should provide intellisense to the developer).